README.md

```diff
@@ -1,23 +1,136 @@
+  C. The `PackageValidationBaselineVersion` value should match the same value as the unconditioned `VersionPrefix`. This property represents the package version that ApiCompat should use to compare the next package version with, to ensure the APIs are fully compatible.
+8. Add the csprojs to the base sln file.
+9. Delete dead source code. For example, code protected by preprocessor directives that are not relevant anymore due to the specified framework being out of support.
+10. Build the whole repo using the base `build.cmd|sh` script.
+11. Turn `<IsPackable>` to `true`, then run `dotnet pack` to see if APICompat / PackageValidation complain, in which case you'll have to address the issues. When done, set it back to `false`.
+12. Squash all the _new_ commits introduced by you in this repo, excluding the migrated commit history.
+13. Submit the PR and tag @dotnet/area-infrastructure-libraries for review.
+
 
 ## How to produce stable package versions
 
 Package versions produced by official builds for this repo will automatically contain a prerelease suffix. This is in order to be able to iterate through changes until we are ready to have a final build when we are ready for a new servicing release. When that time comes, all that is needed in order to produce packages that don't have the prerelease suffix is to manually queue a build in the official pipeline, and set the variable `DotNetFinalVersionKind` to `release`. This will automatically cause Arcade to set the right version for the package, as well as use a dedicated NuGet feed to push the final build assets (in order to avoid potential version clashes).
 
+
 ## How to service a library
 
-The default build will automatically build all of the libraries that have been added to the repo. That said, given this repository is used to service packages, and you won't always want to service all of the packages at the same time, packages for libraries are not generated by default. To enable a library to produce a package and be serviced, set the `<IsPackable>` property to `true` in the library's project file.
+The default build will automatically build all of the libraries that have been added to the repo. That said, given this repository is used to service packages, and you won't always want to service all of the packages at the same time, packages for libraries are not generated by default. To enable a library to produce a package and be serviced:
+
+1. Set the `<IsPackable>` property to `true` in the library's project file.
+2. Set the value of the `<VersionPrefix>` property to the value of the other property with the same name but conditioned to `'$(IsPackable)' == 'true'`.
+3. Bump the minor version number of the `VersionPrefix` property conditioned with `'$(IsPackable)' == 'true'">` property to the next value.
```

Comment:
I'm pretty sure that @ViktorHofer envisioned this happening by a tool that we run as part of branding, not done by the person editing the project.

The person editing the project only needs to toggle IsPackable.

I think he envisioned a process that we run after every release to bump all the versions and clear IsPackable.

Reply:
Correct. Basically a PR like https://github.com/Azure/azure-sdk-for-net/pull/42933/files without the CHANGELOG.md change but with a PackageVersion, AssemblyVersion (if defined) and PackageValidationBaselineVersion rev.

By default the rev updates the patch version only. Reving to a higher major/minor version should be an explicit change, not handled by automation.